src/index.ts

```diff
@@ -0,0 +1,12 @@
+import * as policy from './policy';
+
+
+
+const policyObject = {
+    validate: (policy as any).validate,
+    mutate: (policy as any).mutate,
+};
```

Comment:
We need to ensure that one of these functions are implemented based on the policy type.